### PR TITLE
Updated blazars and supernovae

### PR DESCRIPTION
**Pull request by aryank2240**
updated the article on blazars and supernovae

### CODE REVIEW

File: theory/energetic sources/blazars.md [hunk 1]
```diff
@@ -7,3 +7,19 @@ nav_order: 3
...
+
+### Properties
+
+Blazars have the same properties as a [quasar](./quasars.html) but vary in some aspects like emission spectrum, polarisation, brightness, etc.
+
+Blazars have very high energy outputs that can rival the luminosity of billions of stars on its own. Their energy output can reach up to 10^48 ergs. They also exhibit extreme variability in terms of brightness often changing on timescales from minutes to years. Just like stars blazars also release flares which have an extreme amount of radiation and energy. In 2006 a blazar named PKS 2155-304 released a blazar flare that had a level of luminosity equivalent to the total energy output by 10,000 Milky Way galaxies combined
```
Comment (Jadit19) [inline theory/energetic sources/blazars.md:23]: Use `$$10^{48}$$` for mathjax compatibility

File: theory/energetic sources/supernovae.md [hunk 2]
```diff
@@ -7,3 +7,22 @@ nav_order: 4
 ---
 
 ## Supernovae
+
+A supernova is a powerful explosion that marks the final stage of a star's life. As the star approaches its death, its core begins to collapse under immense pressure. This collapse forces protons and electrons to combine, forming neutrons. The star's outer layers are then expelled into space in a spectacular explosion.
+
+Supernovae release huge amounts of energy. The amount of energy released can go up to 10^51 ergs.
```
Comment (Jadit19) [inline theory/energetic sources/supernovae.md:13]: Use `$$10^{51}$$` for mathjax compatibility

File: theory/energetic sources/supernovae.md [hunk 3]
```diff
@@ -7,3 +7,22 @@ nav_order: 4
...
+Supernovae are the sole reason for heavy elements in our universe. It provides the right conditions for the nucleosynthesis of these elements.
+
+### Types of Supernova
+
+
+1. Type II Supernova:
+When a massive star(Mass more than 8 solar masses) collapses under its own gravity to explode. This explosion is the result of a sudden shockwave formed by the rebound of the collapsing outer layers of the star after they collide with the newly formed dense core.
+
+2. Type I Supernova:
+When a dwarf star in its [binary system](../special%20stars/binary%20stars.html) accretes material from its companion star it can increase its mass beyond the Chandrasekhar limit.After exceeding the limit the star becomes unstable and it triggers a thermonuclear explosion resulting in a supernova.
```
Comment (Jadit19) [inline theory/energetic sources/supernovae.md:27]: Shouldn't Type I supernova be mentioned before Type II?

File: theory/energetic sources/supernovae.md [hunk 4]
```diff
@@ -7,3 +7,22 @@ nav_order: 4
 ---
 
 ## Supernovae
+
```
Comment (Jadit19) [inline theory/energetic sources/supernovae.md:10]: Please format the files before opening a PR. In VS Code, you can make use of the shortcut `Ctrl + Shift + I`

**Review by Jadit19 — CHANGES_REQUESTED** [review]

File: theory/energetic sources/supernovae.md (hunk 2 above)
Comment (aryank2240) [reply theory/energetic sources/supernovae.md]: fixed it

File: theory/energetic sources/supernovae.md (hunk 3 above)
Comment (aryank2240) [reply theory/energetic sources/supernovae.md]: changed the order

**Pull request merged**